src/lib/wpcli-versions.ts

```diff
@@ -5,6 +5,10 @@ import { executeWPCli } from '../../vendor/wp-now/src/execute-wp-cli';
 import getWpCliPath from '../../vendor/wp-now/src/get-wp-cli-path';
 
 export async function updateLatestWPCliVersion() {
+	// @todo remove before merge
+	if ( process.env.STUDIO_IMPORT_EXPORT === 'true' ) {
+		return;
+	}
```

Comment:
Until the new version of `wp-cli` is released with SQLite support, we could consider marking this PR as Not Ready to Merge. Alternatively, we could serve a custom version and update [the download link](https://github.com/Automattic/studio/blob/b883003368f21d840b60f191a39a40db50925746/scripts/download-wp-server-files.ts#L21-L24). WDYT?

Reply:
One approach could be to avoid downloading `wp-cli` in line 17 when the feature flag is enabled. And ensure that the bundled version is the needed one by serving it from a GitHub fork of `wp-cli` or including it in the Studio app repo:
```
	{
		name: 'wp-cli',
		description: 'WP-CLI phar file',
		url: 'https://raw.githubusercontent.com/<REPO>/wp-cli.phar',
		destinationPath: path.join( WP_SERVER_FILES_PATH, 'wp-cli' ),
	},
```